### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by denysLavrienov**
This PR creates a login page, which contains login, password fields and link to the registration page. 
To route between pages was decided to create the `Router` class.

Created components:

* `LoginForm` - stands for login form;
* `Router`- stands for router, that accepts **from**-page and produces the **to**-page;
* `Page404` - stands for error page when page cannot be found.

### CODE REVIEW

File: app/component/login-form.js [hunk 1]
```diff
@@ -21,27 +20,20 @@ export class Form {
...
-      password.errorField = FormValidator.validatePassword(password.value, secondPassword.value);
-    });
-
-    file.onChange((event) => {
-      if (file.inputFiles[0] !== undefined) {
-        file.displayImage();
-      } else {
-        file.errorField = 'No file chosen!';
-      }
+      password.errorField = FormValidator.validatePassword(password.value);
```
Comment (dkalinin) [inline app/component/login-form.js]: Render method should not call any validator. It is responsible for creating DOM and components.

File: app/component/login-form.js [hunk 2]
```diff
@@ -21,27 +20,20 @@ export class Form {
...
     const password = new InputField(form, 'password', 'password', 'Password: ');
-    const secondPassword = new InputField(form, 'second-password', 'password', 'Repeat password: ');
-    const button = new Button(form, 'Register', 'register');
+    const button = new Button(form, 'Login', 'register');
+
+    const referenceToRegister = document.createElement('span');
+    referenceToRegister.innerHTML = `<br>Not registered? <a href = "#/registration">Register</a>`;
+    form.appendChild(referenceToRegister);
 
     button.onClick((event) => {
```
Comment (dkalinin) [inline app/component/login-form.js]: It's not a responsibility of render method.

**Comment by dkalinin** [comment]
@denysLavrienov is this PR going to be fixed?

**Comment by denysLavrienov** [comment]
> @denysLavrienov is this PR going to be fixed?

It so happened that these comments were fixed in the `MVC-implementation` branch before your review.

**Comment by dkalinin** [comment]
> > @denysLavrienov is this PR going to be fixed?
> 
> It so happened that these comments were fixed in the `MVC-implementation` branch before your review.

Please merge then.

**Pull request merged**